nginx.conf

```diff
@@ -16,6 +20,23 @@ http {
 
     ssl_certificate /etc/letsencrypt/live/dev.telescope.cdot.systems/fullchain.pem;
     ssl_certificate_key /etc/letsencrypt/live/dev.telescope.cdot.systems/privkey.pem;
+    ssl_session_timeout 1d;
+    ssl_session_cache shared:MozSSL:10m;  # about 40000 sessions
+    ssl_session_tickets off;
+
+    ssl_protocols TLSv1.2 TLSv1.3;
+    ssl_ciphers ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384;
+    ssl_prefer_server_ciphers off;
+
+    # HSTS
+    add_header Strict-Transport-Security "max-age=63072000" always;
+
+    # OCSP stapling
+    ssl_stapling on;
+    ssl_stapling_verify on;
+
+    # verify chain of  trust of OCSP response using Root CA and Intermediate certs
+    ssl_trusted_certificate /path/to/root_CA_cert_plus_intermediates;
```

Comment:
This should probably be:

```
ssl_trusted_certificate /etc/letsencrypt/live/dev.telescope.cdot.systems/chain.pem;
```

to use the files we get with let's encrypt.

Reply:
Yeah, I had no idea where certbot puts our certs and was in the process of trying to find that

Reply:
If you update this, the rest is good.

One thing I notice with this file is that we'll have to have different versions for staging and prod, since we hard code the `dev.*` prefix into a lot of stuff here.  Should this file be called `nginx-staging.conf`?  cc @manekenpix

Reply:
Yes, I think renaming this to `nginx-staging.conf` will avoid conflicts in the future.

Reply:
renamed file